app_lstu_fr.md

```diff
@@ -1,4 +1,4 @@
-# <img src="/images/lstu_logo.svg" width="80px" alt="logo de LSTU"> LSTU
+# <img src="/images/lstu_logo.svg" heigth="80px" alt="logo de LSTU"> LSTU
```

Comment:
```suggestion
# <img src="/images/lstu_logo.svg" height="80px" alt="logo de LSTU"> LSTU
```

Reply:
Désolé pour cette coquille, ma PR n'était pas très propre. Je ferai plus attention et testerai avant mes fiches, la prochaine PR.
D'ailleurs les fiches mentionnent plutôt width, mais ça fait tout petite sur certains logos longs comme Wordpress :  https://yunohost.org/#/app_wordpress_en/preview
Je pense que le height serait un meilleur choix, non ?
Il fonctionne très bien sur les logos "carrés" : 
https://yunohost.org/#/app_dokuwiki_en/preview

Reply:
Oui en effet, c'est vrai que du coup on peut uniformiser avec `height` plutôt que `width`

Reply:
@webafrancois : Pas besoin de te confondre d'excuses, les typos ça arrive tout le temps chez tout le monde et c'est pour ça qu'on essaye de faire des reviews ;)

Reply:
À propos de ce mécanisme de suggestion, puisque j'ai l'impression que ce n'est pas évident pour tout le monde : c'est un truc intégré dans Github pour permettre facilement de proposer des (petites) modifs quand on fait des reviews. 

Tu peux valider la suggestion en cliquant sur "Commit suggestion" (ou la même chose en fr..). Ou bien lorsqu'il y en a beaucoup, tu peux aussi "Add suggestion to batch" mais seulement depuis l'onglet "Files" (par exemple ici : https://github.com/YunoHost/doc/pull/1281/files pour cette PR). Une fois que tu as fait ça pour toutes les suggestions que tu veux, tu peux commit d'un seul coup toutes les suggestions (ça évite d'avoir 72 commits pour 72 typo)